app/src/main/java/org/oppia/android/app/player/exploration/ExplorationFragmentPresenter.kt

```diff
@@ -56,18 +70,47 @@ class ExplorationFragmentPresenter @Inject constructor(
   fun handleViewCreated() {
     val profileDataProvider = profileManagementController.getProfile(retrieveArguments().profileId)
     profileDataProvider.toLiveData().observe(
-      fragment,
-      { result ->
-        val readingTextSize = retrieveArguments().readingTextSize
-        if (result is AsyncResult.Success && result.value.readingTextSize != readingTextSize) {
-          selectNewReadingTextSize(result.value.readingTextSize)
-
-          // Since text views are based on sp for sizing, the activity needs to be recreated so that
-          // sp can be correctly recomputed.
-          fragment.requireActivity().recreate()
+      fragment
+    ) { result ->
+      val readingTextSize = retrieveArguments().readingTextSize
+      if (result is AsyncResult.Success && result.value.readingTextSize != readingTextSize) {
+        selectNewReadingTextSize(result.value.readingTextSize)
+
+        // Since text views are based on sp for sizing, the activity needs to be recreated so that
+        // sp can be correctly recomputed.
+        fragment.requireActivity().recreate()
+      } else {
+        if (result is AsyncResult.Success) {
+          showSpotlights(result.value.numberOfLogins)
         }
       }
-    )
+    }
+  }
+
+  private fun showSpotlights(numberOfLogins: Int) {
+    val explorationToolbar =
+      fragment.requireActivity().findViewById<View>(R.id.exploration_toolbar) as Toolbar
+    explorationToolbar.forEach {
+      if (it is ImageButton) {
+        // this toolbar contains only one image button, which is the back navigation icon
```

Comment:
```suggestion
        // This toolbar contains only one image button, which is the back navigation icon.
```

Ditto everywhere else in the PR (starting capitalization & ending punctuation).

Reply:
done